addon/components/docs-viewer/x-main/component.js

```diff
@@ -23,20 +23,23 @@ export default Component.extend({
   editCurrentPageUrl: computed('router.currentRouteName', function() {
     let path = this.get('router.currentRouteName');
     if (!path) {
-      // `routing` doesn't exist for old ember versions via ember-try
+      // `router` doesn't exist for old ember versions via ember-try
       return;
     }
+
     path = path.replace(/\./g, '/');
 
-    if (path === 'docs/api/class') {
-      let params = getOwner(this).lookup('route:application').paramsFor('docs.api.class');
-      let klass = dasherize(params.class_id.replace(/-.+$/g, ''));
-      let path = `pods/${path}`;
+    if (path === 'docs/api/item') {
```

Comment:
`docs/api/class` was what the original documentation had, but no longer matches what the guide suggests.